packages/legacy-browser-alert/src/index.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+
+const ieRegexes = [
+    /Trident\/7\.0.*rv\:([0-9\.]+).*\).*Gecko$/,
+    /MSIE\s([0-9\.]+);.*Trident\/[4-7].0/,
+    /MSIE\s(7\.0)/
+];
+
+type Props = {
+  locale?: 'ko' | 'en';
+  sx?: React.CSSProperties;
+  onClose?: () => void;
+}
+
+const AlertComponent = ({ locale = 'ko', sx, onClose }: Props) => {
+  const userAgent = window?.navigator?.userAgent;
+  const isIE = ieRegexes.some(regex => regex.test(userAgent));
+  if (!isIE) {
+    return null;
+  }
+
+  const containerStyle = {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '100%',
+    padding: '1rem 0',
```

Comment:
```suggestion
    padding: '1rem 0.5rem',
```

브라우저 사이즈 쪼그라들었을 때 좌우 여유공간 주면 좋을것 같아요!